configure.ac

```diff
@@ -128,12 +135,16 @@ AS_IF([test "x$ax_sqlite3" != "xyes"],
   ]
 )
 
-dnl Compile the bundled sqlite if there is no system one installed
-AC_MSG_CHECKING(which sqlite3 to use)
-AS_IF([test "x$ax_sqlite3" = "xyes"],
-      [AC_MSG_RESULT([system])
-       PACKAGE_LIBS_PRIVATE="$PACKAGE_LIBS_PRIVATE -lsqlite3"],
-      [AC_MSG_RESULT([bundled])])
+AS_IF([test "x$static_executables" != "xyes"],
```

Comment:
Making the fallback sqlite3 check contingent on building static executables isn't correct regardless of anything else in this PR. It's possible that someone is doing a static build but has no static sqlite3 library installed.

Reply:
There is no other option to disable sqlite3 as a dependency at this moment.

If that is the preferred route happy to go that way. If not, it does not appear to be directly needed for many of the tools. 

Please come with solutions, not with judgement.

Reply:
Have you looked at PR #3046? That contains a change which turns off the use of dlopen in the bundled sqlite3.

I think you're taking all this too personally.

Reply:
No I have not looked at https://github.com/sleuthkit/sleuthkit/pull/3046 given that this is a newer PR. Remove the linux static test, get the PR working and merged and I can update this PR.

> I think you're taking all this too personally.

You are derailing the conversation. This also is a judgement.